client/state/posts/selectors.js

```diff
@@ -313,7 +314,7 @@ export function getEditedPost( state, siteId, postId ) {
  */
 export function getPostEdits( state, siteId, postId ) {
 	const { edits } = state.posts;
-	return get( edits, [ siteId, postId || '' ], null );
+	return normalizeEditedPost( get( edits, [ siteId, postId || '' ], null ) );
```

Comment:
Should we be applying the normalization in `getEditedPost` instead of here? I don't know that it makes much difference, but the `edits` object only contains revisions, not the full object.